grails-app/domain/au/org/emii/aatams/Tag.groovy

```diff
@@ -29,7 +29,7 @@ class Tag extends Device implements Embargoable {
 
     static constraints = {
         project(nullable:true)
-        expectedLifeTimeDays(nullable:true)
+        expectedLifeTimeDays(nullable:false)
         sensors (validator: { sensors ->
```

Comment:
It will be good to add few unit tests checking the following scenarios.

1. Tag creation pass, when expectedLifeTimeDays value is passed
1. Tag creation fails when expectedLifeTimeDays value is not passed
1. Tag creation fails when expectedLifeTimeDays value is 2147483648 or greater
1. Tag creation fails when expectedLifeTimeDays value is (-2147483648 or less) or (less than 0)